dashboard/app/main.go

```diff
@@ -43,6 +44,7 @@ func initHTTPHandlers() {
 	http.Handle("/bug", handlerWrapper(handleBug))
 	http.Handle("/text", handlerWrapper(handleText))
 	http.Handle("/admin", handlerWrapper(handleAdmin))
+	http.Handle("/delete-bisection", handlerWrapper(handleDeleteBisection))
```

Comment:
This should go into admin section in app.yaml.

Also need a test that tests that (1) it's admin only, (2) works as expected and don't corrupt database.

Reply:
1) done by making it an admin action 2) added